src/button.component.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export default function Button(props) {
+  const {
+    children,
+    disabled = false,
+    loading = false,
+    className,
+    ...remainingProps
+  } = props;
+  const background =
+    disabled || loading ? "opacity-50 bg-secondary" : "bg-warning";
+  return (
+    <button
+      className={`mb-8 font-bold py-2 px-4 rounded ${background} ${className}`}
```

Comment:
This will put `undefined` as a css class when className is not provided. Easiest way might be to add default value to props.